src/d3a/models/market/two_sided_pay_as_clear.py

```diff
@@ -29,6 +34,11 @@ def __init__(self, time_slot=None, area=None,
                  notification_listener=None, readonly=False):
         super().__init__(time_slot, area, notification_listener, readonly)
         self.state = MarketClearingState()
+        self.sorted_bids = []
+        self.sorted_pac_offers = []
```

Comment:
Ah no, I just see that this is "heritage" code. Meaning this has been like this sins the PAC was implemented. Will try to change it anyway, ok @faizan2590 ?

Reply:
Sorry @faizan2590 : YES, it needs to be named like this (or at least not sorted_offers) because `sorted_offers` is a property by the Market base class. OK?

Reply:
even better, you don't even have to resort the already sorted_offers. Does it makes sense?

Reply:
You are right! Changing it!